wordpress/wp-content/plugins/sil-dictionary-webonary/include/configuration.php

```diff
@@ -220,8 +220,24 @@ function save_configurations()
 			}
 
 		}
-		update_option("noSearch", $noSearchForm);
+		update_option("noSearch", $noSearchForm);	
```

Comment:
Looks like there is trailing white-space.

Reply:
Fixed